app/invocation/invocation_model.tsx

```diff
@@ -388,23 +390,60 @@ export default class InvocationModel {
     return this.optionsMap.get("remote_instance_name") ?? "";
   }
 
+  getCompressor() {
+    if (this.isCacheCompressionEnabled()) {
+      return build.bazel.remote.execution.v2.Compressor.Value.ZSTD;
+    }
+    return build.bazel.remote.execution.v2.Compressor.Value.IDENTITY;
+  }
+
   getDigestFunction() {
     const digestFnName = this.optionsMap.get("digest_function");
     if (!digestFnName) {
-      return undefined;
+      return build.bazel.remote.execution.v2.DigestFunction.Value.SHA256;
```

Comment:
this seems error-prone, since we used to let the server infer it from digest length?  is it actually guaranteed that the option will be set for old-style digest functions?

Reply:
Everything you said is right--I was just wondering aloud if it's ever possible for bazel to have used SHA1 (for example) without the flag being set this way.  It's true that this has to work for blake3 URLs to work properly (i.e., if there even is a bug here, it already exists for blake3), so this is fine.